amundsen-kube-helm/templates/helm/templates/amundsen-service.yaml

```diff
@@ -18,7 +18,6 @@ spec:
     - port: 5001
       name: {{ .Chart.Name }}-{{ .Values.search.serviceName }}-{{ .Values.environment }}-http
       targetPort: 5001
-      nodePort: 30001
```

Comment:
I could not find a need to set the node port on a load balancer, but, if someone has a major objection, I can add it in with an optional lookup.